test/views/StartViewSpec.scala

```diff
@@ -16,190 +16,127 @@
+      view.getElementById("section-header").text() mustBe "startPage.title.sectionHeader"
     }
 
-    "display \"Export\" description" in {
-
-      getElementByCss(createView(), "article>div>div>p:nth-child(2)")
-        .text() must be(messages(description))
+    "display header" in {
+      view.getElementById("title").text() mustBe "startPage.title"
     }
 
-    "display list header" in {
-
-      getElementByCss(createView(), "h3").text() must be(messages(listHeading))
+    "display general description" in {
+      view.getElementById("description").text() mustBe "startPage.description"
     }
 
-    "display list with elements" should {
-
-      "first element" in {
-
-        getElementByCss(createView(), "article>div>div>ul>li:nth-child(1)")
-          .text() must be(messages(listItemOne))
-      }
-
-      "second element" in {
-
-        getElementByCss(createView(), "article>div>div>ul>li:nth-child(2)")
-          .text() must be(
-          messages(listItemTwoPreUrl) + " " + messages(listItemTwoUrl) + " " + messages(listItemTwoPostUrl)
-        )
-      }
-
-      "link in second element to \"EORI\" page" in {
-
-        val link =
-          getElementByCss(createView(), "article>div>div>ul>li:nth-child(2)>a")
-
-        link.text() must be(messages(listItemTwoUrl))
-        link.attr("href") must be("http://www.gov.uk/eori")
-      }
-
-      "third element" in {
-
-        getElementByCss(createView(), "article>div>div>ul>li:nth-child(3)")
-          .text() must be(messages(listItemThree))
-      }
-
-      "fourth element" in {
+    "display Contents section" in {
+      view.getElementById("contents").text() mustBe "startPage.contents.header"
 
-        getElementByCss(createView(), "article>div>div>ul>li:nth-child(4)")
-          .text() must be(messages(listItemFour))
-      }
-
-      "fifth element" in {
-
-        getElementByCss(createView(), "article>div>div>ul>li:nth-child(5)")
-          .text() must be(messages(listItemFive))
-      }
-
-      "sixth element" in {
-
-        getElementByCss(createView(), "article>div>div>ul>li:nth-child(6)")
-          .text() must be(messages(listItemSix) + " " + messages(listItemSixUrl))
-      }
-
-      "link in sixth element to \"Commodity codes\" page" in {
-
-        val link =
-          getElementByCss(createView(), "article>div>div>ul>li:nth-child(6)>a")
-
-        link.text() must be(messages(listItemSixUrl))
-        link.attr("href") must be("https://www.gov.uk/trade-tariff")
-      }
-
-      "seventh element" in {
-
-        getElementByCss(createView(), "article>div>div>ul>li:nth-child(7)")
-          .text() must be(messages(listItemSevenUrl) + " " + messages(listItemSeven))
-      }
-
-      "link in seventh element to\"Customs procedure codes\" page" in {
-
-        val link =
-          getElementByCss(createView(), "article>div>div>ul>li:nth-child(7)>a")
+      view.getElementById("contents-element-1").text() mustBe "startPage.beforeYouStart.header"
```

Comment:
Are we interest with structure or just fact that was rendered in cirrect section.

Reply:
As I remember we were testing the structure some time ago, but moved away from that.